lib/modules/device/DeviceService.ts

```diff
@@ -457,21 +459,33 @@ export class DeviceService {
         this.getDeviceModel(device._source.model),
       ]);
 
-      this.checkAlreadyProvidedMeasures(device, asset, measureNames);
+      let updatedMeasureNames: MeasureName[] = [];
+      for (const measure of measureNames) {
+        const type = deviceModel.device.measures.find(
+          (deviceMeasure) => deviceMeasure.name === measure.device
+        ).type;
```

Comment:
Are we sure that the provided measure name for the device exists in the device model definition?

Reply:
I can check that before that. If it's null, I need to add the measure without type or I can find it in another place (on the asset and/or device) ?

Reply:
If the user want to link the asset and the device with a measure name that doesn't exists then we should throw an error (my bad I forgot this case I think)

Reply:
In the case of implicit linking (in generateMissingAssetMeasureNames) it seems undeclared measures are not created. So if I throw an error, it could be a breaking change ?

Reply:
Yes you are right :+1: 

Then we should check only if implicit linking was not specified